src/pages/task/logs/logTypes/TaskEventLogLine.tsx

```diff
@@ -1,16 +1,14 @@
 import React from "react";
 import styled from "@emotion/styled/macro";
 import { format } from "date-fns";
+import { paths } from "constants/routes";
 import { TaskEventLogEntry } from "gql/generated/types";
-import { getUiUrl } from "utils/getEnvironmentVariables";
 
 const FORMAT_STR = "MMM d, yyyy, h:mm:ss aaaa";
 
 export const TaskEventLogLine: React.FC<TaskEventLogEntry> = (props) => {
   const { timestamp, eventType, data } = props;
-  const hostLink = (
-    <a href={`${getUiUrl()}/host/${data.hostId}`}>{data.hostId}</a>
-  );
+  const hostLink = <a href={`${paths.host}/${data.hostId}`}>{data.hostId}</a>;
```

Comment:
we should use `Link` from `import { Link } from "react-router-dom";` since we are now routing within the application instead of navigating to the legacy ui